assigner/commands/init.py

```diff
@@ -39,6 +39,7 @@ def guess_semester():
 
 @config_context
 def init(conf, _):
+    conf['backend'] = prompt("Backend to use (gitlab)", "gitlab")
```

Comment:
Let's leave this choice out right now -- we'd have to rearchitect `init` anyway if we introduced a new backend since the next couple of config lines are GitLab-specific.